src/System.Private.Interpreter/src/Internal/Runtime/Interpreter/ILImporter.Interpreter.cs

```diff
@@ -123,12 +132,19 @@ private void ImportBreak()
 
         private void ImportLoadVar(int index, bool argument)
         {
-            throw new NotImplementedException();
+            if (argument)
```

Comment:
These should have TODOs and throw NotImplementedException so that they are easy to trace down.

Reply:
Done